pages/api/events.js

```diff
@@ -0,0 +1,39 @@
+import {promisify} from 'util';
+import stream from 'stream';
+import got from 'got';
```

Comment:
Used `got` so we can stream the response straight to the client

This avoids running additional JSON parse and stringify on the server, as they can be slow for large strings of json

It "pipes" the result of the elvanto request straight into the nextjs response

```js
await pipeline(
    elvantoRequest,
    response
  );
```

Reply:
Cool